packages/circuit-ui/components/Avatar/Avatar.stories.tsx

```diff
@@ -33,25 +33,28 @@ Base.args = {
   src: 'https://source.unsplash.com/EcWFOYOpkpY/200x200',
   variant: 'object',
   size: 'yotta',
+  alt: 'A cup of coffee on a table',
```

Comment:
I also added the missing (required) alt text to stories. I missed this in the Avatar PR because we're not running tsc on stories

Reply:
I think we are 🤔 However, TypeScript doesn't know that `<ComponentName>.args` corresponds to the component props.

Reply:
It's excluded here 😛 

https://github.com/sumup-oss/circuit-ui/blob/890b77624714a7e4dcba0349f264b7628bb16feb/packages/circuit-ui/tsconfig.json#L44

I think the blocker is the `mdx` imports in stories (for docs), enabling TS would fail until we solve this somehow

Reply:
Ah right, thanks 😃